mainnet-contracts/src/WithdrawalManagerStorage.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+/**
+ * @title WithdrawalManagerStorage
+ * @author Puffer Finance
+ * @custom:security-contact security@puffer.fi
+ */
+abstract contract WithdrawalManagerStorage {
+    struct Withdrawal {
+        // 96bits
+        uint128 pufETHAmount; 
+        uint128 pufETHToEthExchangeRate;
+        address recipient; //160bits
+    }
+
+    struct WithdrawalBatch {
```

Comment:
we should add warnings for these structs as well.

DO NOT CHANGE, REORDER, REMOVE EXISTING STORAGE VARIABLES ...